app/src/main/java/com/example/parkingsystemjava/MainActivity.java

```diff
@@ -2,11 +2,26 @@
 import androidx.appcompat.app.AppCompatActivity;
 import android.os.Bundle;
 
+import com.example.parkingsystemjava.databinding.ActivityMainBinding;
+import com.example.parkingsystemjava.mvp.model.ParkingModel;
+import com.example.parkingsystemjava.mvp.presenter.ParkingPresenter;
+import com.example.parkingsystemjava.mvp.view.ParkingView;
+
 public class MainActivity extends AppCompatActivity {
+    private ActivityMainBinding binding;
+    private ParkingPresenter presenter;
```

Comment:
Here you need to use the contract type, and then when you instantiate the atributte yo use the clases

Reply:
done